config/default.json

```diff
@@ -7,7 +7,8 @@
   "redis": {
     "url": "redis://127.0.0.1:6379",
     "secret": "defaultsecret",
-    "useSSL": "false"
+    "useSSL": "false",
+    "timeout": 1200
```

Comment:
1200 in seconds it's 20 mins but the copy says 30

Reply:
We need to add a line of content to the bottom of each step warning the user of the 20 minute timeout. As it stands the timeout is very abrupt and without warning. 
"This page will time out in 20 minutes if there is no activity. This is to protect your data."

Going forward https://tools.hmcts.net/jira/browse/SSCS-3180 will address the issue by posing a question "Want more time to continue" with yay/nay buttons.

Reply:
@paulgain Okie dokie! Will add that later.